### PR TITLE
Web: Add Hr Element

### PR DESCRIPTION
**Pull request by hfhbd**

### CODE REVIEW

File: web/core/src/jsMain/kotlin/androidx/compose/web/elements/Elements.kt [hunk 1]
```diff
@@ -358,6 +359,18 @@ fun Footer(
     )
 }
 
+@Composable
+fun Hr(
+    attrs: AttrBuilderContext<Tag.Hr> = {},
+    content: ContentBuilder<HTMLHRElement>? = null
```
Comment (eymar) [inline web/core/src/jsMain/kotlin/androidx/compose/web/elements/Elements.kt:365]: Is there a need for `content` for <hr> tag?

Comment (hfhbd) [reply web/core/src/jsMain/kotlin/androidx/compose/web/elements/Elements.kt:365]: You are right. It is even forbidden.

Comment (hfhbd) [reply web/core/src/jsMain/kotlin/androidx/compose/web/elements/Elements.kt:365]: BTW Br is also not allowed to contain a content https://developer.mozilla.org/en-US/docs/Glossary/Empty_element

Comment (eymar) [reply web/core/src/jsMain/kotlin/androidx/compose/web/elements/Elements.kt]: Good catch! Thanks. 
Would you like to remove content from Br element in this PR as well?  That would be helpful :)

Comment (eymar) [reply web/core/src/jsMain/kotlin/androidx/compose/web/elements/Elements.kt]: oh I see you added an issue for EmptyElements. Thanks! Then I accept this PR.

**Pull request merged**